lighthouse-core/aggregator/aggregate.js

```diff
@@ -37,12 +37,8 @@ class Aggregate {
    */
   static _getTotalWeight(expected) {
     const expectedNames = Object.keys(expected);
-    let weight = expectedNames.reduce((last, e) => last + expected[e].weight, 0);
-    if (weight === 0) {
-      weight = 1;
-    }
-
-    return weight;
+    let totalWeight = expectedNames.reduce((last, e) => last + (expected[e].weight || 0), 0);
```

Comment:
In my opinion `(expected[e].weight || 0)`is a bit hard to read. maybe add a comment that you default to 0